.env

```diff
@@ -0,0 +1,13 @@
+SECRET_KEY='%s@@0ml3qt&%ge3sl=+^f269qdu0x3v5u5idw(=10cf$scqk=@'
+
+PSQL_DB_HOST="pgdb"
+PSQL_DB_PORT=5432
+#PSQL_DB_HOST="localhost"
```

Comment:
Should these go in a .env.local file?

Reply:
I want to remove the `.env` file from repo. I have added it git ignore file but somehow it pushed.